[terra-alert] Add Unsatisfied and Unverified Types (#2941)
cerner/terra-core: packages/terra-alert/src/terra-dev-site/doc/alert/Alert.1.doc.mdx

# Terra Alert

The Terra Alert component is a notification banner that can be rendered in your application when there is information that you want to bring to the user's attention.

> The Alert component supports a number of built-in notification types that render with pre-defined colors and icons that help the user understand the severity and meaning of the notification. The support alert types and recommended uses are:

Reviewer: ```suggestion
The Alert component supports a number of built-in notification types that render with pre-defined colors and icons that help the user understand the severity and meaning of the notification. The supported alert types and recommended uses are:
```

> - `alert` - recommended to display a critical notifications for current emergencies or situations of high severity that need to be addressed promptly

Reviewer: ```suggestion
- `alert` - recommended to display a critical notification for current emergencies or situations of high severity that need to be addressed promptly
```

> - `alert` - recommended to display a critical notification for current emergencies or situations of high severity that need to be addressed promptly

Reviewer 2: we mention these are recommendations above; should we drop "recommended" from these descriptions then?
```suggestion
- `alert` - displays a critical notification for current emergencies or situations of high severity that need to be addressed promptly
```

Reviewer 2: ah, for consistency, if we make this change it should be carried through to the other descriptions

- `error` - recommended to provide feedback of a negative system status such as failures, inabilities, or limitations
- `warning` - recommended to show moderately urgent notifications that are cautionary in nature and forecast potential risk

> - `unsatisfied` - recommended to provide the user that an unsatisfied or incomplete action needs to be addressed

Reviewer: ```suggestion
- `unsatisfied` - recommended to show the user that an unsatisfied or incomplete action needs to be addressed
```

> - `unverified` - recommended to provide feedback that information from outside systems have been included but have yet to be verified by the user

Reviewer: ```suggestion
- `unverified` - recommended to provide feedback that information from outside systems has been included but has yet to be verified by the user
```

- `advisory` - recommended to provide the user with guidance and/or advice for completing an action

> - `information` - recommended to provide general awareness and information that can possibly lead to an action for something that but has no associated severity.

Reviewer: ```suggestion
- `information` - recommended to provide general awareness and information that can possibly lead to an action for something that has no associated severity.
```
